lib/ansible/modules/network/panos/panos_import.py

```diff
@@ -47,13 +50,20 @@
     category:
         description:
             - Category of file uploaded. The default is software.
+            - See API > Import section of the API reference for category options.
         default: software
     file:
         description:
             - Location of the file to import into device.
     url:
         description:
             - URL of the file that will be imported to device.
+    validate_cert:
```

Comment:
To be consistent with other modules, this should be validate_certs, not validate_cert.